nose2/sphinxext.py

```diff
@@ -88,6 +88,31 @@ def document(self, rst, plugin):
         rst.append(u'', AD)
 
     def add_config(self, rst, config, plugin):
+        # add docs on enabling non-default plugins
+        plugin_modname = plugin.__module__
+        if plugin_modname not in plugins.DEFAULT_PLUGINS:
+            self.headline(rst, u'Enable this Plugin')
+            rst.append(
+                u'This plugin is built-in, but not loaded by default.', AD)
+            rst.append(u'', AD)
+            rst.append(
+                u'Even if you specify ``always-on = True`` in the '
+                u'configuration, it will not run unless you also enable it. '
+                u'You can do so by putting the following in a '
+                u':file:`unittest.cfg` or :file:`nose2.cfg` file',
```

Comment:
Would it be possible to link here to more information about where this file is searched?

Reply:
I would have to dig through to find the relevant code as well.

I'm going to merge this PR now, but feel free to open additional PR or open discussion on enhancing this text.